lib/models/currency.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const Model = require('../model')
+const BOOL_FIELDS = []
+const FIELDS = {
+  id: 0,
+  name: 1
```

Comment:
I prefer `id` to `abbreviation` since it's shorter, `abbr` could work if you insist :slightly_smiling_face: `name` is better than `fullName` imo